example/stories/index.js

```diff
@@ -27,7 +27,7 @@ stories.add('with all knobs', () => {
 
   const bold = boolean('Bold', false);
   const selectedColor = color('Color', 'black');
-  const favoriteNumber = number('Favorite Number', 42);
+  const favoriteNumber = select('Favorite Number', { 'Seven': 7, 'Forty-two': 42, 'Nine thousand': 9000 }, 42);
```

Comment:
I think it would be better to add a new line, keeping the simple "number" example.